index.html

```diff
@@ -24,41 +26,164 @@
+      /></a>
+      <ul id="navbar-h2">
+        <li><a href="./about.html">About</a></li>
+        <li><a>Program</a></li>
+        <li><a>Join</a></li>
+        <li><a>Sponsor</a></li>
+        <li><a>News</a></li>
+        <li><button>CC Campaign</button></li>
+      </ul>
+    </div>
     <section id="section1">
+      <nav class="menu-responsive">
+        <div class="x-section">
+          <img src="./images/x-icon.png" alt="" />
+        </div>
+        <ul class="navbar-menu-responsive">
+          <li>
+            <a href="./about.html" class="items-menu-responsive">About</a>
+          </li>
+          <li><a href="#" class="items-menu-responsive">Program</a></li>
+          <li><a href="#" class="items-menu-responsive">Join</a></li>
+          <li><a href="#" class="items-menu-responsive">Sponsor</a></li>
+          <li><a href="#" class="items-menu-responsive">News</a></li>
+          <li><button>CC Campaign</button></li>
+        </ul>
+      </nav>
       <div id="main-banner">
-        <img id="hamburg" src="./images/hamburg.png" alt="" />
+        <img id="hamburg" src="./images/menu.png" alt="" />
         <div id="main-content">
-          <h4>"Hello! Sharing world"</h4>
-          
-          <h1 id="text-image">CREATIVE COMMONS GLOBAL SUMMIT <br> 2015</h1>
-          
+          <h4>Don't miss these trips!</h4>
+
+          <h1 id="text-image">
+            FIND MUSIC FESTIVALS ALL OVER THE WORLD<br />
+            2023
+          </h1>
         </div>
         <div id="first-parraf-banner">
           <p>
-            A joyful celebration believing in the value of openness and sharing,
-            creating a positive change with people from over 80 countries is
-            taking place in October, in Korea.
+            The best festivals around the world. Discover festivals and get
+            tickets. Find festivals by country. The largest platform for buying
+            and selling tickets around the world. Quick purchase. All over the
+            world.
           </p>
         </div>
         <div id="second-parraf-banner">
-          <h4>2015.10.15(THU) ~ 16(FRI)</h4>
-          <h6>@ National Museum of Karea, Art Center Nabi and more</h6>
+          <h4>2023.01.15(THU) ~ 16(FRI)</h4>
+          <h6>@ Tomorrowland, Lollapalooza and more</h6>
         </div>
       </div>
-      <div id="main-program"></div>
-      <div id="Featured Speaker"></div>
+      <div id="main-program">
+        <h3>Our Core Services Program</h3>
+        <hr class="line" />
+        <div class="cards-container">
+          <div class="cards-main-program">
+            <img
+              class="img-cards-main-program"
+              src="./images/program_icon_01.png"
+              alt=""
+            />
+            <h5>Hire meet and greet</h5>
+            <p>
+              Don't miss the opportunity to get to know your favorite artist up
+              close together with other fans.
+            </p>
+          </div>

```

Comment:
- [ ] For each card, I would suggest using the [`<article>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) tag as a container instead of just a `<div>`, as each one of those cards represents a self-contained composition of content! 🤓 🔧